Update eosjs

kjg opened the pull request:
Eosjs v22 is now out, we can at least update to eosjs v21.x?    v21.x switches to the elliptic cryptography library, adds WebAuthn Support, and contains other security fixes

kjg commented:
@michaeljyeates Would you consider merging this, or are there some changes I can make? Thanks!

michaeljyeates commented:
> @michaeljyeates Would you consider merging this, or are there some changes I can make? Thanks!

Sorry, are you able to update the pull request based on the latest code (just a yarn.lock conflict) and i will merge

kjg commented:
Hi @michaeljyeates, I've rebased and I've updated Eosjs to v22 now so that it matches the version [used in the updated waxjs](https://github.com/eosdac/ual-wax/blob/1c207264eed2a50988d5c774fa32f3d3b2bfcb7e/yarn.lock#L38)
